src/utils/svgHelper.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+import { html } from 'lit-element';
+
+import { SvgIcon } from '../components/mgt-person-card/mgt-person-card';
+
+/**
+ * returns an svg
+ * @param svgIcon defined by name
+ * @param color hex value
+ */
+export function getSvg(svgIcon: SvgIcon, color: string) {
+  switch (svgIcon) {
+    case 0:
```

Comment:
for readability, you should use `case SvgIcon.Phone:`

Reply:
Updated:

3c1d087